ConnectorNavisworks/ConnectorNavisworks/Other/SelectionBuilder.cs

```diff
@@ -48,24 +48,26 @@ public void GetFromFilter()
   {
     switch (_filter.Slug)
     {
-      case FilterTypes.Manual:
+      case FilterTypes.MANUAL:
         _uniqueModelItems.AddRange(GetObjectsFromSelection());
         break;
 
-      case FilterTypes.Sets:
+      case FilterTypes.SETS:
         _uniqueModelItems.AddRange(GetObjectsFromSavedSets());
         break;
 
-      case FilterTypes.Views:
+      case FilterTypes.VIEWS:
         _uniqueModelItems.AddRange(GetObjectsFromSavedViewpoint());
         break;
+      default:
```

Comment:
Being picky I'd prefer a line space between the break of the previous case and the default case and generally between break and cases but maybe that's coming soon.